src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  position: absolute;
+  gap: $gap;
+  max-width: calc($columns * $cell-size + $columns * $gap + 2 * $padding
```

Comment:
between 7 cells you have 6 gaps so you need to make 7*$cell-size + 6*$gap + 2*$padding to defined width